packages/transit-vehicle-overlay/src/index.story.js

```diff
@@ -144,7 +146,7 @@ function rectangles(popup = true) {
       updateTrackedVehicle(vehicle, isTracked);
       if (!isTracked)
         // clear recenter coords, so map recenters on repeated clicks of same marker
-        recenter(null, 0, 0);
+        recenter(null, 0, true, 0);
```

Comment:
Same as https://github.com/opentripplanner/otp-ui/pull/255/files#r648383383, I am not sure to understand the change in this function call.